### ISSUE TITLE
ui command

### ISSUE THREAD

**Issue by Xgamer6** [BambooPM/SimpleNPC#1]
Please add a /npc ui command to manage the NPCS faster!

**Comment by pandaa-be** [comment]
good idea, in the next weeks I will release a new version of MobsLite and SimpleNPCs.
